docs/source/configuration/traffic-shaping.mdx

```diff
@@ -9,7 +9,7 @@ the traffic between the client and router and between the router and subgraphs.
 
 ## Configuration
 
-To enable traffic shaping, add the `traffic_shaping` plugin to your [YAML config file](./overview/#yaml-config-file), like so:
+The `traffic_shaping` plugin is always enabled with default values, if you want to override default values you should add `traffic_shaping` to your [YAML config file](./overview/#yaml-config-file), like so:
```

Comment:
Good call, @garypen. It would be great if we could have a table of defaults somewhere on this page that could be linked like so:

```suggestion
By default, the `traffic_shaping` plugin is enabled with [preset values](#link-to-default-values). To override presets, add `traffic_shaping` to your [YAML config file](./overview/#yaml-config-file) like so:
```